packages/graphql-typescript-definitions/src/index.ts

```diff
@@ -350,48 +494,37 @@ export class Builder extends EventEmitter {
 
     return document;
   }
+}
 
-  private getProjects() {
-    return getGraphQLProjects(this.config);
-  }
-
-  private getGlobs() {
-    return this.getProjects().reduce<string[]>((globs, project) => {
-      return globs.concat(
-        project.includes.map((filePath) =>
-          getGraphQLFilePath(this.config, filePath),
-        ),
-      );
-    }, []);
-  }
-
-  private getSchemaPaths() {
-    return getGraphQLSchemaPaths(this.config);
-  }
-
-  private removeDocumentForFile(filePath: string) {
-    const project = this.config.getConfigForFile(filePath);
-
-    if (project) {
-      const documents = this.documentMapByProject.get(project.projectName);
-
-      if (documents) {
-        documents.delete(filePath);
-      }
-    }
-  }
+function getUniqueFilePaths(filePaths: string[]) {
```

Comment:
I don't think this is necessary, there should be no duplicate file paths unless we don't trust `glob`